src/__tests__/Hug.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import 'jest-styled-components'
```

Comment:
Line 16 is using this.  I'll remove the setup file for now, but it may be useful later.  Since almost all of our tests are using `jest-styled-components`, by including it in a test-setup file, we can remove the import from all the other tests.  I'll save that for later though, because it should be done in a separate PR.